### PR TITLE
dev-tools: add instructions to update flake to 0.2

### PR DESCRIPTION
**Pull request by jost-s**

### CODE REVIEW

**Review by ThetaSinner — APPROVED** [review]

File: src/pages/get-building/upgrade-holochain.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+::: coreconcepts-intro
+For existing hApps that have been written for Holochain version 0.1, these are steps to upgrade to Holochain version 0.2.
+:::
+
+### Steps to upgrade Nix flake from Holochain version 0.1 to 0.2
+
+!!! note
+The following steps only apply to Nix flakes that have been either generated by the Holochain scaffolding tool
+or follow the recommended format printed under [Dev Tools Setup](../install-advanced){target=_blank}.
```
Comment (zippy) [inline src/pages/get-building/upgrade-holochain.md:15]:
```suggestion
or follow the recommended format as described in [Dev Tools Setup](../install-advanced){target=_blank}.
```

File: src/pages/get-building/upgrade-holochain.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+!!!
+
+In the root folder of the hApp that should be upgraded, execute the following command in a terminal:
```
Comment (zippy) [inline src/pages/get-building/upgrade-holochain.md:18]:
```suggestion
In the root folder of the hApp to be upgraded, execute the following command in a terminal:
```

**Review by zippy — APPROVED** [review]
The alternative set of instructions doesn't include the `nix flake update`  which I think is crucial no?

**Comment by jost-s** [comment]
> The alternative set of instructions doesn't include the `nix flake update` which I think is crucial no?

You're right, it's crucial. I'll leave only the text replacement command in and move the other two the result section.

**Pull request merged**